### ISSUE TITLE
Different results inferenced by the same tflite from windows and android

### ISSUE THREAD

**Issue by PGogo** [google/mediapipe#3222]
I build up my facial emotion recognition module plugged into face_mesh_mobile pipeline, and make a subgraph of face_landmark_front_gpu.pbtxt base on face_landmark_gpu.pbtxt with the same input_streams ("IMAGE:image" and "ROI:roi"). I compile my android aar with the above modifications. Then I train my custom facial emotion recognition tflite which works fine in my tflite inference demo (using tensorflow python api without mediapipe) on windows. Finally, it turns out bad inference result on android.
I've got a visualization on the input tensor of tflite and ensure the face detection area to be adaptive to the tflite.
What do I miss? I wonder is there any possible that the same tflite could output results with greate difference between these two platform?
Here is my subgraph (tflite could be provided by email if necessary), and Thanks for any help!!!

##################################################
type: "FaceEmotionGpu"

# GPU image. (GpuBuffer)
input_stream: "IMAGE:image"
# ROI (region of interest) within the given image where a face is located.
# (NormalizedRect)
input_stream: "ROI:initial_roi"

output_stream: "EMOTIONS_PRED:emotions_pred_logits"

# Calculate size of the image.
node {
  calculator: "ImagePropertiesCalculator"
  input_stream: "IMAGE_GPU:image"
  output_stream: "SIZE:image_size"
}

# Expands and shifts the rectangle that contains the face so that it's likely
# to cover the entire face.
node {
  calculator: "RectTransformationCalculator"
  input_stream: "NORM_RECT:initial_roi"
  input_stream: "IMAGE_SIZE:image_size"
  output_stream: "roi"
  options: {
    [mediapipe.RectTransformationCalculatorOptions.ext] {
      scale_x: 0.6666
      scale_y: 0.6666
      square_long: true
    }
  }
}

# Transforms the input image into a 96x96 tensor.
node: {
  calculator: "ImageToTensorCalculator"
  input_stream: "IMAGE_GPU:image"
  input_stream: "NORM_RECT:roi"
  output_stream: "TENSORS:em_input_tensors"
  options: {
    [mediapipe.ImageToTensorCalculatorOptions.ext] {
      output_tensor_width: 96
      output_tensor_height: 96
      output_tensor_float_range {
        min: 0.0
        max: 1.0
      }
      gpu_origin: TOP_LEFT
    }
  }
}

# Runs a TensorFlow Lite model on GPU that takes an image tensor and outputs a
# vector of GPU tensors representing, for instance, detection boxes/keypoints
# and scores.
node {
  calculator: "InferenceCalculator"
  input_stream: "TENSORS:em_input_tensors"
  output_stream: "TENSORS:emotions_pred_tensor"
  options: {
    [mediapipe.InferenceCalculatorOptions.ext] {
      model_path: "mediapipe/modules/face_landmark/face_emotion.tflite"
      #
      delegate: { gpu { use_advanced_gpu_api: true } }
    }
  }
}

# Converts the emotion prediction logit tensors into a float vector that represents the confidence
# score of the facial emotion.
node {
  calculator: "TensorsToFloatsCalculator"
  input_stream: "TENSORS:emotions_pred_tensor"
  output_stream: "FLOATS:emotions_pred_logits"
}

**Comment by sureshdagooglecom** [comment]
Hi @PGogo ,
Could you please provide steps to reproduce this issue.

**Comment by PGogo** [comment]
> Hi @PGogo , Could you please provide steps to reproduce this issue.

Thank you for your help. Here are my steps.

1. Train my custom model and conver to tflite.
2. In order to insert my module into the pipeline of face_mesh_mobile, I make a subgraph refer to face_landmark_gpu.pbtxt and also modify several related pbtxt
3. Every thing works fine running on android, except the prediction of the tflite. Since it gives a lower accuracy  than running on windows(using tensorflow python api without mediapipe)

It seems that my tflite output different inference results between android and windows environments. Is it possible? And how could I make them exactly the same?
